tests/topology/expected/vlan-vrrp.yml

```diff
@@ -274,6 +279,35 @@ nodes:
       vlan:
         mode: irb
         name: red
+    - bridge_group: 2
+      gateway:
+        id: -2
+        ipv4: 172.16.1.254/24
+        protocol: vrrp
+        vrrp:
+          group: 1
+      ifindex: 6
+      ifname: Vlan1001
+      ipv4: 172.16.1.2/24
+      name: VLAN blue (1001) -> [s2]
+      neighbors:
+      - gateway:
+          anycast:
```

Comment:
Interfaces get cleaned up, but not neighbors

Reply:
Correct. Have to mention that "neighbors" are not a reliable data source, they are more of a hint of who's around. Changes to interface data after the link transformation are usually not reflected in _neighbors_ list (and that applies to a bunch of modules and plugins)